e2e/pages/contract-boost-calculator.spec.ts

```diff
@@ -166,3 +166,89 @@ test('calcs an 8-tok', async ({ page }) => {
 
 	for (const match of outputs) await expect(out).toContainText(match);
 });
+
+test('calcs a 5-tok', async ({ page }) => {
+	// prettier-ignore
+	await page.getByRole('radio', { name: '-token (Benson)' }).click();
+	let outputs = [
+		[/runs out after/i, /10min/],
+		[/ge cost/i, /10,400/],
+		[/online/i, /625.585M/],
+		[/offline/i, /1.877B/],
+		[/hab space/i, /11.34B/],
+		[/time to fill/i, /∞/],
+	];
```

Comment:
`// prettier-ignore` directives only apply to the immediately following AST node, so it needs to be above the `let outputs`, which should be formatted as a nice table, like the one above it:

```suggestion
	await page.getByRole('radio', { name: '-token (Benson)' }).click();

	// prettier-ignore
	let outputs = [
		[/runs out after/i, /10min/   ],
		[/ge cost/i,        /10,400/  ],
		[/online/i,         /625.585M/],
		[/offline/i,        /1.877B/  ],
		[/hab space/i,      /11.34B/  ],
		[/time to fill/i,   /∞/       ],
	];
```